### PR TITLE
Document the fact that Firefox needs explicit user input to execute an extension on file:// URLs

### PR DESCRIPTION
**Pull request by Mogztter**
Ref #256 

@aivarannamaa @jasontibbitts @mojavelinux is it clear enough ?

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -41,8 +41,14 @@ image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.p
...
-WARNING: On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
-Please note that this issue can also occured on other websites with restictive CSP.
+[WARNING]
+====
+On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
+
+In addition, Firefox does not allow access to `file://` URLs unless explicit user input.
+It means that you must type or copy the `file://` URL into your browser address bar and then press "Enter".
+There's an open issue in the https://bugzilla.mozilla.org/show_bug.cgi?id=1266960[Firefox bug tracker] to fix this limitation.
+====
```
Comment (mojavelinux) [inline README.adoc]: I'd make this a little bit more clear. Here's a draft.

> Firefox does not trigger the extension when navigating to a `file://` from a link. Instead, you have to enter the `file://` URL directly into the address bar of the browser. If you do navigate to a `file://` URL from a link, simply highlight the address bar and press "Enter" to force the extension to load. There's an open issue...

File: README.adoc [hunk 2]
```diff
@@ -41,8 +41,14 @@ image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.p
  2. Open local or remote .ad, .adoc, .asc, .asciidoc files in Firefox
  3. Enjoy!
 
-WARNING: On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
-Please note that this issue can also occured on other websites with restictive CSP.
+[WARNING]
+====
+On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
```
Comment (mojavelinux) [inline README.adoc]: Ventilate this prose?

Comment (Mogztter) [reply README.adoc]: as in, one sentence per line in the AsciiDoc document or think we should break down the second sentence ?

Comment (Mogztter) [reply README.adoc]: I did both, let me know if it's better

File: README.adoc (hunk 1 above)
Comment (Mogztter) [reply README.adoc]: Done 👍

File: README.adoc [hunk 3]
```diff
@@ -41,8 +41,16 @@ image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.p
...
 
-WARNING: On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
-Please note that this issue can also occured on other websites with restictive CSP.
+[WARNING]
+====
+On Firefox, pages viewed directly from GitHub may not render properly.
+This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the Content Security Policies (CSP).
+The restrictive CSP sent by GitHub are applied to the content inserted by the WebExtension.
+
+In addition, Firefox does not trigger the extension when navigating to a `file://` from a link.
```
Comment (mojavelinux) [inline README.adoc]: On minor rewording request.

> In additiona, the extension does not automatically trigger in Firefox when navigating to a `file://` from a link.

That puts the correct subject first.

Comment (Mogztter) [reply README.adoc]: Done :+1:

**Comment by mojavelinux** [comment]
Perfect!

**Comment by jasontibbitts** [comment]
Yes, that would certainly have eliminated my confusion.  Thanks!

**Pull request merged**